.travis.yml

```diff
@@ -33,9 +33,11 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.7.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.*@dev
+      env: SYMFONY_VERSION=2.8.*
     - php: 5.6
-      env: SYMFONY_VERSION="3.0.x-dev" DEPENDENCIES="dev" COMPOSER_FLAGS="--prefer-stable"
+      env: SYMFONY_VERSION=3.0.*
+    - php: 5.6
+      env: SYMFONY_VERSION=3.1.*@dev
```

Comment:
Why? It was not the case for 3.0-dev: https://github.com/sonata-project/SonataIntlBundle/blob/master/.travis.yml#L39

Errors should be resolved instead.